File: benchmarks/static/corpus/go/vuln.go
package p
import ("os/exec"; "os"; "crypto/md5"; "database/sql")
func h(u, up string, db *sql.DB) {
    exec.Command("sh", "-c", u)   // EXPECT GF-404
    os.ReadFile(up)               // EXPECT GF-405
    md5.New()                     // EXPECT GF-422
    db.Query("SELECT * FROM t WHERE x=" + u)  // EXPECT GF-419
}

File: benchmarks/static/corpus/java/Vuln.java
class Vuln { void h(String u, java.io.ObjectInputStream ois, java.sql.Statement st) throws Exception {
    Runtime.getRuntime().exec("sh -c " + u);              // EXPECT GF-404
    st.executeQuery("SELECT * FROM t WHERE x=" + u);      // EXPECT GF-419
    Object o = ois.readObject();                          // EXPECT GF-421
    java.security.MessageDigest.getInstance("MD5");       // EXPECT GF-422
    String apiKey = "AKIAsecretvalue1234";                // EXPECT GF-429
} }
